src/modules/material-requests/services/material-requests.service.ts

```diff
@@ -78,7 +77,7 @@ export class MaterialRequestsService {
 
 		if (!isEmpty(type)) {
 			where.type = {
-				_eq: MaterialRequestTypes[type],
+				_eq: MaterialRequestType[type],
```

Comment:
we don't need to resolve this type again, since type will already be the correct value of the enum.
This code assumes the type is a key of the enum.